packages/yoga/src/Popover/web/Popover.jsx

```diff
@@ -49,16 +69,11 @@ function Popover({
 
       <PopoverButton
         {...(a11yId && { 'aria-describedby': a11yId })}
-        onMouseEnter={handleShowPopover}
-        onMouseLeave={handleHidePopover}
-        onTouchStart={handleShowPopover}
-        onTouchEnd={handleHidePopover}
-        onClick={event => event.preventDefault()}
-        onKeyDown={event => {
-          if (event.key === 'Enter') {
-            setIsPopoverOpen(current => !current);
-          }
-        }}
+        ref={ref}
+        onMouseEnter={hover ? handleShowPopover : undefined}
+        onMouseLeave={hover ? handleHidePopover : undefined}
+        onClick={!hover ? toggleOpenPopover : undefined}
+        onKeyDown={event => event.key === 'Enter' && toggleOpenPopover}
```

Comment:
toggleOpenPopover should be called. Also keep the code as before:
```suggestion
        onKeyDown={event => {
          if (event.key === 'Enter') {
             toggleOpenPopover();
         }
        }}
```

Becase if `event.key === 'Enter'` is false, the function will return `false`. But it should be `void`

Reply:
@flavia-moraes don't forget this change please :)

Reply:
This implementation will not work, unless you use an arrow function before it, something like this
`() => toggleOpenPopover()`
The updated code is:
` onKeyDown={event =>
            event.key === 'Enter' ? toggleOpenPopover : undefined
          }`